docs/README.md

```diff
@@ -149,27 +156,27 @@ and instrument your .NET application using the provided PowerShell module.
 Example usage (run as administrator):
 
 ```powershell
-# Download and import the module
+# Download the module
 $module_url = "https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/v0.5.1-beta.3/OpenTelemetry.DotNet.Auto.psm1"
 $download_path = Join-Path $env:temp "OpenTelemetry.DotNet.Auto.psm1"
 Invoke-WebRequest -Uri $module_url -OutFile $download_path
+
+# Import the module to use its functions
 Import-Module $download_path
 
 # Install core files
 Install-OpenTelemetryCore
 
-# Setup IIS instrumentation
-Register-OpenTelemetryForIIS
-
-# Setup your Windows Service instrumentation
-Register-OpenTelemetryForWindowsService -WindowsServiceName "MyServiceName" -OTelServiceName "MyServiceDisplayName"
-
-# Setup environment to start instrumentation from the current PowerShell session
+# Setup the instrumentation for the current PowerShell session
 Register-OpenTelemetryForCurrentSession -OTelServiceName "MyServiceDisplayName"
 
-# Get current installation location
-Get-OpenTelemetryInstallDirectory
+# Run your application with instrumentation
+dotnet run
```

Comment:
I know that `dotnet run` is not currently working

@pjanotti Should we keep it or change the example to use some executable? E.g.

```suggestion
.\MyNetApp.exe
```

Reply:
I will explicitly call out the current issues with `dotnet` CLI and suggest to create a shell to run the instrumented version. Basically: build and run `dotnet` CLI on shell __1__ then set up the environment for automatic instrumentation on shell __2__. A warning icon, plus, a link to the bug seems worthwhile at the moment.

Reply:
I will address it in a separate PR.